src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -169,9 +169,28 @@ public void aliasTest() throws IOException {
 
     @Test
     public void groupByTest() throws Exception {
-
         JSONObject result = executeQuery(String.format("SELECT COUNT(*) FROM %s/account GROUP BY gender",
-                TEST_INDEX_ACCOUNT));
+            TEST_INDEX_ACCOUNT));
+        assertResultForGroupByTest(result);
+    }
+
+    @Test
+    public void groupByUsingTableAliasTest() throws Exception {
+        JSONObject result = executeQuery(String.format("SELECT COUNT(*) FROM %s/account a GROUP BY a.gender",
```

Comment:
I think we should use Locale.root here and elsewhere, or StringUtils

Reply:
Yes, it seems all test cases in this IT don't use our own `StringUtils`. I guess there are also a lot similar cases in other tests. Do you think it's okay to create a maintenance issue for tracking for now?

Reply:
Opened issue for tracking: https://github.com/opendistro-for-elasticsearch/sql/issues/223

Reply:
Thanks for creating the issue!